blueprints/addon/index.js

```diff
@@ -106,7 +106,7 @@ module.exports = {
   },
 
   mapFile: function(file, locals) {
-    var result = this._super.mapFile.call(this, file, locals);
+    var result = this._super(file, locals);
```

Comment:
Should be updated to `this._super.mapFile.apply(this, arguments);` (we should only use bare `this._super()` in `init`).